src/form-host/save-contact.js

```diff
@@ -110,8 +110,8 @@ const prepareAndAttachSiblingDocs = function(doc, original, siblings) {
                 original[fieldName]._id === value) {
       doc[fieldName] = original[fieldName];
     } else {
-      // TODO: 
-      // 
+      // TODO:
```

Comment:
Wish I hadn't touched it. I think it just happened because I deleted trailing spaces.
In the future, I'm hoping more of our enketo code will be in a `shared-lib` and we can kill all this stuff.